tests/test_cases/test_vhdl_zerovector/Makefile

```diff
@@ -0,0 +1,29 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Language of toplevel module
+TOPLEVEL_LANG = vhdl
+
+ifneq ($(filter $(SIM), questa modelsim),)
+    # VHDL language version
+    VCOM_ARGS += -2008
+    # simulation time precision
+    SIM_ARGS += -t ns
+endif
+
+ifneq ($(filter $(SIM),ius xcelium),)
+    SIM_ARGS += -v93
+endif
+
```

Comment:
```suggestion
```
Already handled [here](https://github.com/cocotb/cocotb/blob/9fe131d93f31438d032307f70096fac797af78e0/cocotb/share/makefiles/simulators/Makefile.ius#L79).

Reply:
Handled for IUS but not Xcelium, so this must stay in.
(I removed this from the Makefile,xcelium so that the user is able to select what VHDL version is used without the simulator seeing conflicting options)